src/translib/transformer/xfmr_system.go

```diff
@@ -176,8 +175,8 @@ func getSystemCpu (idx int, cpu Cpu, syscpus *ocbinds.OpenconfigSystem_System_Cp
         log.Infof("syscpus.NewCpu failed")
         return
     }
-    ygot.BuildEmptyTree(syscpu)
-    syscpu.Index = &index
+   ygot.BuildEmptyTree(syscpu)
```

Comment:
This to me is a workaround. Please find the actual issue.

Reply:
Done